connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java

```diff
@@ -28,7 +28,7 @@ public class StandaloneConfig extends WorkerConfig {
      * <code>offset.storage.file.filename</code>
      */
     public static final String OFFSET_STORAGE_FILE_FILENAME_CONFIG = "offset.storage.file.filename";
-    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store offset data in";
+    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store source task offsets";
```

Comment:
To remain consistent with the description for the `offset.storage.topic`, we should probably use the term "connector" instead of "task":
```suggestion
    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store source connector offsets";
```

Reply:
Yes, I like consistency. Thanks.